### PR TITLE
CSV-File-getting-From-Py-issue#17

### PR DESCRIPTION
**Pull request by kaochaiyakarn**
This PR is added the python script file which works on this following list.
The python script list ->

1. Getting the data from the Probot(STRING file).
2. Changing the data type from String file to Dict with the JSON library
3. Crawling the data from the Dict follow this order
     3.1 The branch name of the PR
     3.2 The Commit Hash
     3.3 The clone Url
     3.4 The Repo Name
4. Execution all the data which we get in JIT.py (Prototype from Tui) (Terminal execution)
5. Export the metric in CSV file form. This will be in the Test file, so it can add or update. It makes the 
    python code have some function which I provide but didn't use at all.

In conclusion, the CSV file will be stored in the python script folder after the PR created.

Other files which aren't .py are the log of the Probot.

This PR already fixes add the .gitignore which remove the unnecessary file in the Probot template.
Sorry for mistake PR before.

### CODE REVIEW

File: probotPRV1/stale-master/pythonScript/CommitMetricOutput.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Commit Hash,NS,ND,NF,Entropy,LA,LD,LT,FIX,NDEV,AGE,NUC,EXP,REXP,SEXP,BUGLABEL
```
Comment (jirayusjiar) [inline probotPRV1/stale-master/pythonScript/CommitMetricOutput.csv]: Please update the README file to provide metric description in the next pull request.

File: probotPRV1/stale-master/pythonScript/retriveData.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    clone_cmd = 'git clone ' + clone_url
+    os.system(clone_cmd)
+    
+    # Start JIT
+    chane_branch = 'git checkout ' + PR_branch
+    os.chdir(repo_name)
+    os.system(chane_branch)
+    os.system("git branch")
+    os.chdir("..")
+    command = 'python jit.py ' + repo_name
```
Comment (jirayusjiar) [inline probotPRV1/stale-master/pythonScript/retriveData.py]: Just curious. So this line is "python jit.py repo_name ALL" which extracts all commits from the PR_branch correct?

Comment (kaochaiyakarn) [reply probotPRV1/stale-master/pythonScript/retriveData.py]: From Line 44, the code will change to the branch which is the brach that the PR use to work.
Therefore, line 47 will extract all commit in the PR instead of all commit in Master

Comment (jirayusjiar) [reply probotPRV1/stale-master/pythonScript/retriveData.py]: Good!

File: probotPRV1/stale-master/pythonScript/.vscode/settings.json [hunk 3]
```diff
@@ -1,4 +1,4 @@
 {
-    "python.pythonPath": "/Users/chaiyakarn/miniconda3/bin/python",
+    "python.pythonPath": "/usr/local/bin/python3",
```
Comment (jirayusjiar) [inline probotPRV1/stale-master/pythonScript/.vscode/settings.json]: Will this work on all machines?, e.g., an AWS server and Windows.
Do python has a relative path to import that works for all OS and machines?
